aws/policy/application-services.yaml

```diff
@@ -16,6 +16,7 @@ Statement:
       - cloudformation:ListResourceRequests
       - cloudformation:ListResources
       - cloudformation:UpdateResource
+      - cloudformation:DescribeType
```

Comment:
Could you make sure this change is ordered alphabetically? It starts to get difficult to manage if things get all mixed in.